app/views/pages/rollover.html.erb

```diff
@@ -7,19 +7,29 @@
     </h1>
 
     <p class="govuk-body">
-      All your courses, locations and details have been copied over
-      from the current recruitment cycle (<%= current_recruitment_cycle_period_text %>). You can now
-      update them for <%= next_recruitment_cycle_period_text %>.
+      Courses for the next cycle (<%= next_recruitment_cycle_period_text %>) will be published on 1 September 2021.
```

Comment:
The "1 September 2021" date was a placeholder for testing. We shouldn't publish this without confirming the actual date first.

Reply:
> Is there more to be updated?

@defong Among the cards in this sprint on the Candidate board, nothing that relates to that partial. Maybe a question for @paulrobertlloyd / @frankieroberto ?

Reply:
Can we make this date a constant somewhere? Then here we can format that date as above, and for the ‘Publish in October’ button we can use that same date, but formatted as the month. Make sense? @malcolmbaig @defong

Reply:
Should we change it to 'in October' then?

Reply:
@paulrobertlloyd @B-Bonner will we not have a date confirmed by the time rollover starts? Feels like we ought to be able to tell providers this!?

Reply:
Being confirmed today, I’m told.

Reply:
5 October 2021.

Reply:
Will move that into a constant.